### DISCUSSION TITLE
Plugin Stylesheet and Dark and Light Theme

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#8280] [Q&A]
In Shiraz I use Bootstrap 5.3.3 which supports light and dark color schemes. Bootstrap uses  `[data-bs-theme=light]` and `[data-bs-theme=dark]` in its stylesheets to define colors.

How can I setup Shiraz stylesheets or other plugins to follow TiddlyWiki color-scheme?

I thought to conditionally activate dark/light color schemes in plugins like below

```
<%if [{$:/palette}get[color-scheme]match[light]] %>
    <$transclude light-color-scheme stylesheets />
<%else%>
    <$transclude dark-color-scheme stylesheets />
<%endif%>

```

Is there any better solution by TiddlyWiki to allow plugin developer support dark/light themes?

**Comment by pmario** [comment]
There is a pending PR - See: 

- https://github.com/Jermolene/TiddlyWiki5/pull/7830
